integration-tests/vertx-graphql/src/test/java/io/quarkus/vertx/graphql/it/VertxGraphqlTest.java

```diff
@@ -37,7 +31,7 @@ public void testGraphQlQuery() {
 
     @Test
     public void testWebSocketSubProtocol() throws Exception {
-        HttpClient httpClient = vertx.createHttpClient();
+        HttpClient httpClient = Vertx.vertx().createHttpClient();
```

Comment:
This new Vert.x instance must be closed.

Reply:
Well spotted, I'll add it later on.

Reply:
done